### DISCUSSION TITLE
Reduce source and receiver imprint in traveltime inversion

### DISCUSSION THREAD

**Discussion by sminize** [gimli-org/pyGIMLi#374] [Q&A]
Hallo everyone,
i want to do travel time inversions. I have the problem that I always get a strong source and receiver imprint in the inversion result.
When my setup is for example the following (on the top are the sources):
![image](https://user-images.githubusercontent.com/23382227/156519668-ae64d5b6-ae19-4794-bba2-9d8d4c6e0407.png)
I get following inversion result:

![image](https://user-images.githubusercontent.com/23382227/156519827-1d5bcb84-5e7e-4ad7-aa52-db15f1036f2f.png)

The model for calculating the travel times was homogeneous, but in the middle was a circular insert with a lower velocity.
I used the following parameters for the inversion:
`invmodel = tt.invert(data, mesh=mesh, secNodes=3, lam=2500, zWeight=1,
                     useGradient=False, verbose=False,blockyModel=True,robustdata=False)`
Is there a way to reduce the source and receiver imprint in the inversion?

All the best and thank you for your help.

Simon

**Comment by halbmy** [comment]
I would start turning `blockyModel=False` since this could amplify artifacts. Moreover, you can play with regularization, e.g. using `zWeight=0.3` to enhance vertical contrasts (the artifacts are mostly horizontal ones).

Generally, the velocity contrasts seems to be large and the regularization strength likewise. I am afraid that with the given experimental design (low number of shots, only vertical rays, a probably small anomaly) you will not be able to achieve a good resolution.
